scripts/loadAssets.py

```diff
@@ -0,0 +1,109 @@
+# File: loadAssets.py
+#
+# Author: Craig Cogdill
+
+import json
+import time
+from elasticsearch import Elasticsearch
+from os import listdir
+from os.path import isfile, join, splitext
+from util import ElasticsearchUtil
+from util import Utility
+from util import Logger
+es = Elasticsearch(max_retries=5, retry_on_timeout=True)
```

Comment:
I think this could become a class as well. Same logic as the above, too many global variables. It would be cleaner as a class. You can do this later.